src/components/breadcrumbs/breadcrumbs.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { screen, render } from "@testing-library/react";
+import StyledBreadcrumbs from "./breadcrumbs.style";
+import Breadcrumbs from "./breadcrumbs.component";
+import StyledCrumb from "./crumb/crumb.style";
+
+test("Breadcrumbs component", () => {
+  render(
+    <StyledBreadcrumbs>
```

Comment:
You should render the React components rather than the styled components
```
<Breadcrumbs>
      <Crumb text="Crumb 1" href="#" />
      <Crumb text="Crumb 2" href="#" />
      <Crumb text="Crumb 3" href="#" />
      <Crumb text="Crumb 4" href="#" isCurrent />
    </Breadcrumbs>
```